_posts/2022-09-12-sql-search-relevance-selective-aggregation-and-window-functions-in-OpenSearch.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "SQL search relevance, selective aggregation, and window functions in OpenSearch"
+authors:
+- dai-chen
```

Comment:
short name looks like it should be "daichen"

Reply:
Adjusted